### DISCUSSION TITLE
Do you have the option to produce captions limited by line length from transcriptions?

### DISCUSSION THREAD

**Discussion by team-deepgram** [deepgram/community#408] [General help] [answered]
Hi @404 (Luke) , I was just chekcing out your documentation, have a doubt, do you guys have an option to limit the no of characters to a particular piece of transcription? Like I want to have 3 words max in my transcription objects for a single utterance. Is there any way i achieve it? Thanks.

> [Posted by hassannaftabb](https://discord.com/channels/1108042150941294664/1169193228159959095/1169193228159959095)

**Comment by lukeocodes** [comment, marked as answer]
I'm glad you use Node.js because we released this new open source package from Deepgram only this week

https://github.com/deepgram/deepgram-node-captions

You only need to import the `srt` function from the package, and provide it the response from our API as an object/dict.

```ts
import { srt } from "@deepgram/captions";

const result = srt(deepgram_transcription_result);
```
 
It will then format it into valid SRT text. You can customise the line length, but it defaults to 8 words. Based on the average word length in the english language being 4.8, 8 words on average will fit the standard 16:9 screen for subtitles. This works with and without utterances enabled.

If you'd like to customise the line length, you provide it a second parameter.

```ts
import { srt } from "@deepgram/captions";

const result = srt(deepgram_transcription_result, 12); // 12 words per line
```
